src/services/smartWallet.js

```diff
@@ -301,17 +301,41 @@ class SmartWallet {
     return this.sdk.searchAccount({ address });
   }
 
-  async getAccountPayments(lastSyncedHash: ?string, page?: number = 0) {
+  /**
+   * SDK API call results are sorted descending by creation date
+   * lastSyncedId is used to determine whether this page was already fetched
+   */
+  async getAccountPayments(lastSyncedId: ?number, page?: number = 0) {
```

Comment:
What do we expect to happen if `lastSyncedId` is null? Couldn't we just return right away?

Reply:
if `lastSyncedId` is null, then we will return all the elements we would be able to fetch